lib/pages/scout/usecases/GetScoutAthletesDataUseCase.dart

```diff
@@ -1,30 +1,50 @@
 import 'package:ax_dapp/pages/scout/models/AthleteScoutModel.dart';
+import 'package:ax_dapp/repositories/CoinGeckoRepo.dart';
 import 'package:ax_dapp/repositories/SportsRepo.dart';
 import 'package:ax_dapp/repositories/subgraph/SubGraphRepo.dart';
 import 'package:ax_dapp/service/BlockchainModels/TokenPair.dart';
 import 'package:ax_dapp/service/athleteModels/SportAthlete.dart';
 import 'package:ax_dapp/service/athleteModels/mlb/MLBAthlete.dart';
 import 'package:ax_dapp/util/SupportedSports.dart';
+import 'package:coingecko_api/coingecko_result.dart';
+import 'package:coingecko_api/data/market_data.dart';
 
 class GetScoutAthletesDataUseCase {
+  static const collateralizationMultiplier = 1000;
   final SubGraphRepo graphRepo;
+  final CoinGeckoRepo coinGeckoRepo;
   final Map<SupportedSport, SportsRepo<SportAthlete>> _repos = Map();
   List<TokenPair> allPairs = [];
 
-  GetScoutAthletesDataUseCase({required this.graphRepo, required List<SportsRepo<SportAthlete>> sportsRepos}) {
+  GetScoutAthletesDataUseCase(
+      {required this.graphRepo,
+      required this.coinGeckoRepo,
+      required List<SportsRepo<SportAthlete>> sportsRepos}) {
     sportsRepos.forEach((repo) {
       _repos[repo.sport] = repo;
     });
   }
 
+  Future<double> fetchAxPrice() async {
+    final axPrice;
+    CoinGeckoResult axMarketData = await coinGeckoRepo.getAxPrice();
+    List<MarketData> axDataByCurrencty =
+        axMarketData.data.marketData.dataByCurrency;
+    axPrice = axDataByCurrencty.firstWhere((axPrice) => axPrice.coinId == 'usd');

```

Comment:
I believe it should be spelled as `axDataByCurrency`

Reply:
yeah, good call